tests/test_build_material.py

```
@@ -0,0 +1,223 @@
+    assert body == "ただの手書きメモ．\n2 行目．"
+
+
+def test_parse_memo_frontmatter_only_has_empty_body() -> None:
+    text = "---\ntags: []\n---\n"
+    front, body = parse_memo(text)
+    assert front["tags"] == []
+    assert body == ""
+
+
+def test_parse_memo_raises_on_malformed_frontmatter() -> None:
+    # インデント不正で YAML パースに失敗する例．無言で本文扱いにせず明示エラーとする
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF003)